source/kubernetes.rst

```diff
@@ -80,15 +80,7 @@ Install Percona server for MongoDB on Kubernetes
 
    More details about secrets can be found in `Users <users.html>`_.
 
-6. Install `cert-manager <https://docs.cert-manager.io/en/release-0.8/getting-started/install/kubernetes.html>`_ if it is not up and running yet, then generate and apply certificates as secrets according to `TLS document <TLS.html>`_.
```

Comment:
Can we say here something like, operator will generate certificates automatically, but you can generate and apply own certificates on this step